src/commands/dev/mod.rs

```diff
@@ -35,14 +45,23 @@ pub async fn dev(
 ) -> Result<(), failure::Error> {
     commands::build(&target)?;
     let server_config = ServerConfig::new(host, ip, port)?;
+    let session_id = get_session_id()?;
+    let preview_id = get_preview_id(target, user, &server_config, &session_id)?;
 
+    thread::spawn(move || socket::listen(session_id));
+
+    let mut rt = Runtime::new()?;
+    rt.block_on(serve(server_config, preview_id))?;
```

Comment:
nit: this is probably from some boilerplate, but generally speaking i prefer we use full words for variable names. s/rt/runtime/g